### PR TITLE
Update: Rewrite AI Gateway Getting Started guide

### PR DESCRIPTION
**Pull request by lena-larionova**
### Description

We've gotten feedback that the AI getting started script is difficult to use as it obfuscates what's actually happening in the setup. This changes the getting started guide to set up a service, route, and plugin manually, then test that the plugin is working. 

Tested this locally, still need to test the Konnect instructions.

I left the AI quickstart script in at the bottom of the page with a bit of info on how to use it, for those people that are still interested. This script is very useful for demos and testing and we don't want to lose it.

### Testing instructions

Preview link: https://deploy-preview-8056--kongdocs.netlify.app/gateway/latest/get-started/ai-gateway/

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

### CODE REVIEW

File: app/_src/gateway/get-started/ai-gateway.md [hunk 1]
```diff
@@ -1,93 +1,287 @@
...
-allowing you to evaluate the core AI provider capabilities before exploring the full suite of AI plugins.
+Kong AI Gateway is a set of [AI plugins](/hub/?category=ai), which can be used by
+installing {{site.base_gateway}} and then by following the documented configuration instructions for each plugin. 
+The AI plugins are supported in all deployment modes, 
+including {{site.konnect_short_name}}, self-hosted traditional, hybrid, and DB-less, and on Kubernetes via the 
+[{{site.kic_product_name}}](/kubernetes-ingress-controller/latest/).
 
-The quickstart script supports deploying {{site.base_gateway}} on {{site.konnect_product_name}} or 
-in traditional mode on Docker only.
+The AI plugins are fully supported by {{site.konnect_short_name}} in both hybrid mode,
```
Comment (Guaris) [inline app/_src/gateway/get-started/ai-gateway.md:16]:
```suggestion
AI plugins are fully supported by {{site.konnect_short_name}} in both hybrid mode, 
```

File: app/_src/gateway/get-started/ai-gateway.md [hunk 2]
```diff
@@ -1,93 +1,287 @@
...
+including {{site.konnect_short_name}}, self-hosted traditional, hybrid, and DB-less, and on Kubernetes via the 
+[{{site.kic_product_name}}](/kubernetes-ingress-controller/latest/).
 
-The quickstart script supports deploying {{site.base_gateway}} on {{site.konnect_product_name}} or 
-in traditional mode on Docker only.
+The AI plugins are fully supported by {{site.konnect_short_name}} in both hybrid mode, 
+and as a [fully managed service](/konnect/gateway-manager/dedicated-cloud-gateways).
+
+You can enable most {{site.base_gateway}} AI capabilities with one of the following plugins:
+* [AI Proxy](/hub/kong-inc/ai-proxy/) plugin (open-source)
```
Comment (Guaris) [inline app/_src/gateway/get-started/ai-gateway.md:20]:
```suggestion
* [AI Proxy](/hub/kong-inc/ai-proxy/): The open source AI proxy plugin.
```

File: app/_src/gateway/get-started/ai-gateway.md [hunk 3]
```diff
@@ -1,93 +1,287 @@
...
+* [AI Proxy Advanced](/hub/kong-inc/ai-proxy-advanced/) (Enterprise): 
+More advanced load balancing, routing, and retries
```
Comment (Guaris) [inline app/_src/gateway/get-started/ai-gateway.md:22]:
```suggestion
* [AI Proxy Advanced](/hub/kong-inc/ai-proxy-advanced/): 
The enterprise version offering more advanced load balancing, routing, and retries. 
```

File: app/_src/gateway/get-started/ai-gateway.md [hunk 4]
```diff
@@ -1,93 +1,287 @@
...
+{% endnavtabs %}
+
+Adding this route entity creates an ingress rule on the `/openai` path. 
+
+{:.note}
+> These examples use the fake URL `http://fake.host.internal` as the upstream URL for the service - but you don't need to replace it with a real one.
+This is because the service upstream URL won't really matter, because after installing the AI Proxy plugin (or AI Proxy Advanced), 
+the upstream proxying destination will be determined dynamically based on your AI Proxy plugin configuration.
+
+### 2. Install an AI Proxy plugin
```
[the remark this answers is not in the document]
Comment (Guaris) [reply app/_src/gateway/get-started/ai-gateway.md:113]: I know you talk about both but I think its okay to mention just one.

File: app/_src/gateway/get-started/ai-gateway.md [hunk 5]
```diff
@@ -1,93 +1,287 @@
...
+## Installing more AI plugins
+
+The AI Proxy and AI Proxy Advanced plugins are able to understand the incoming OpenAI protocol, 
+so you can now further apply deep capabilities like:
+
+* Route to [all other LLMs](/hub/kong-inc/ai-proxy-advanced/) that we support, 
+even the ones that don't natively support the OpenAI specification, as Kong will automatically transform the request. 
+Write once, and use all LLMs.
+* Extract [observability metrics for AI](/gateway/latest/production/monitoring/ai-metrics/).
+* Accelerate traffic with the [AI Semantic cache plugin](/hub/kong-inc/ai-semantic-cache/) plugin.
```
Comment (Guaris) [inline app/_src/gateway/get-started/ai-gateway.md:227]:
```suggestion
* Cache traffic using the [AI Semantic cache plugin](/hub/kong-inc/ai-semantic-cache/) plugin.
```

File: app/_src/gateway/get-started/ai-gateway.md [hunk 6]
```diff
@@ -1,93 +1,287 @@
...
+* Secure traffic with the [AI Prompt Guard](/hub/kong-inc/ai-prompt-guard/) and 
+[AI Semantic Prompt Guard](/hub/kong-inc/ai-semantic-prompt-guard/) plugins.
+* Provide prompt templates with [AI Prompt template](/hub/kong-inc/ai-prompt-template/).
+* Automatically inject system or assistant prompts on every incoming prompt via the
```
Comment (Guaris) [inline app/_src/gateway/get-started/ai-gateway.md:231]:
```suggestion
* Programmatically inject system or assistant prompts to all incoming prompts with the 
```

File: app/_src/gateway/get-started/ai-gateway.md [hunk 7]
```diff
@@ -1,93 +1,287 @@
...
+        "window_size": 60
+      }
+    ]
+  }
+}'
+```
+
+Every other {{site.base_gateway}} plugin can also be used in addition to the AI plugins, for advanced access control, authorization and authentication, security, observability, and more.
+
+## Quickstart script
```
Comment (Guaris) [inline app/_src/gateway/get-started/ai-gateway.md:260]: I think we can delete this from here and throw it in: 
https://deploy-preview-8056--kongdocs.netlify.app/gateway/latest/ai-gateway/

But I don't feel very strongly about it so feel free to keep it.

**Review by Guaris — CHANGES_REQUESTED** [review]

File: app/_src/gateway/get-started/ai-gateway.md [hunk 8]
```diff
@@ -1,93 +1,287 @@
...
+  },
+  "system_fingerprint": "fp_r0bdr52e6e"
+}
+```
+
+And just like that, your traffic is now being properly proxied to OpenAI via {{site.base_gateway}} and its AI plugins!
+
+## Installing more AI plugins
+
+The AI Proxy and AI Proxy Advanced plugins are able to understand the incoming OpenAI protocol,
```
Comment (lena-larionova) [inline app/_src/gateway/get-started/ai-gateway.md:220]:
```suggestion
AI Proxy and AI Proxy Advanced plugins are able to understand the incoming OpenAI protocol.
```

File: app/_src/gateway/get-started/ai-gateway.md (hunk 7 above)
Comment (lena-larionova) [reply app/_src/gateway/get-started/ai-gateway.md:260]: I'm good with that, and I think it would be a better compromise for this whole update.

File: app/_src/gateway/ai-gateway/index.md [hunk 9]
```diff
@@ -214,4 +214,33 @@ AI operations, helping you monitor performance and costs effectively.
...
+This script deploys {{site.base_gateway}} in a local Docker container in traditional mode, or as a data plane instance for {{site.konnect_short_name}}.
+
+This option is for demo purposes only, and is not meant for production use.
+
+{:.note}
+> **Note:**
+> Running this script prompts you for AI Provider API Keys which are used to configure authentication with
+> hosted AI providers. These keys are only passed to the {{site.base_gateway}} Docker container and are 
+> not otherwise transmitted outside the host machine.
+
```
Comment (Guaris) [inline app/_src/gateway/ai-gateway/index.md:231]:
```suggestion
Kong offers an interactive AI quickstart script that launches a demo instance of {{site.base_gateway}} running AI Proxy. 

```sh
curl -Ls https://get.konghq.com/ai | bash
```

The script will run in either traditional mode or as a data plane instance for {{site.konnect_short_name}}. You will be prompted to input an API key to configure authentication with an AI provider. This key will not be exposed outside of the host machine.
```

Comment (Guaris) [reply app/_src/gateway/ai-gateway/index.md:231]: This rendered poorly, let me try again: 


This is what I mean: 

Kong offers an interactive AI quickstart script that launches a demo instance of {{site.base_gateway}} running AI Proxy. 

```sh
curl -Ls https://get.konghq.com/ai | bash
```

The script will run in either traditional mode or as a data plane instance for {{site.konnect_short_name}}. You will be prompted to input an API key to configure authentication with an AI provider. This key will not be exposed outside of the host machine.

File: app/_src/gateway/ai-gateway/index.md [hunk 10]
```diff
@@ -214,4 +214,33 @@ AI operations, helping you monitor performance and costs effectively.
...
+```sh
+curl -Ls https://get.konghq.com/ai | bash
+```
+
+Follow the prompts to set up the AI Proxy plugin with the LLM provider of your choice.
```
Comment (Guaris) [inline app/_src/gateway/ai-gateway/index.md:236]: For some reason the suggestion thing isnt working but I would delete this.

**Review by Guaris — CHANGES_REQUESTED** [review]

**Review by Guaris — APPROVED** [review]
